src/solver/impls/arkode/arkode.cxx

```diff
@@ -213,51 +215,28 @@ int ArkodeSolver::init() {
     throw BoutException("ARKStepCreate failed\n");
   }
 
-  switch (treatment) {
-  case Treatment::ImEx:
-    output_info.write("\tUsing ARKode ImEx solver \n");
-    if (ARKStepSetImEx(arkode_mem) != ARK_SUCCESS) {
-      throw BoutException("ARKStepSetImEx failed\n");
-    }
-    break;
-  case Treatment::Explicit:
-    output_info.write("\tUsing ARKStep Explicit solver \n");
-    if (ARKStepSetExplicit(arkode_mem) != ARK_SUCCESS) {
-      throw BoutException("ARKStepSetExplicit failed\n");
-    }
-    break;
-  case Treatment::Implicit:
-    output_info.write("\tUsing ARKStep Implicit solver \n");
-    if (ARKStepSetImplicit(arkode_mem) != ARK_SUCCESS) {
-      throw BoutException("ARKStepSetImplicit failed\n");
-    }
-    break;
-  default:
-    throw BoutException("Invalid treatment: {}\n", toString(treatment));
-  }
-
```

Comment:
@drreynolds @maggul This looks redundant with the call to `ARKstepCreate` before this, so I've removed the code. Can you double check this is ok?

Reply:
I agree with @Steven-Roberts -- there is no need to call `SetImEx`, `SetExplicit`, or `SetImplicit` in this context.